discoverable-client/src/main/resources/logback.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+    <property resource="application.yml" />
+
+    <turboFilter class="org.zowe.apiml.product.logging.UseridFilter"/>
+    <turboFilter class="org.zowe.apiml.product.logging.ApimlDependencyLogHider"/>
+    <turboFilter class="org.zowe.apiml.product.logging.LogLevelInfoFilter"/>
+    <turboFilter class="ch.qos.logback.classic.turbo.DuplicateMessageFilter">
+        <AllowedRepetitions>0</AllowedRepetitions>
+    </turboFilter>
```

Comment:
I would remove filters which we don't need: dependencyloghider, loglevelinfofilter and duplicatemessagefilter. In fact, i would even take duplicatemessagefilter down from other services if in DEV mode

Reply:
i added the logback mainly because of the duplicate filter so I would leave it here, I can delete the others